lists2safebrowsing.py

```diff
@@ -699,6 +731,7 @@ def main():
     else:
         print('\n\n*** Unable to get branches from shavar-prod-lists repo ***')
 
+    request_review()
```

Comment:
The `request_review()` only works for RS upload. I think we should only request this if RS is enabled. Also, It would be better to move to publish2cloud because RS upload logic is implemented in that.

Reply:
I've updated the function be in publish2cloud.py, but it needs to be called here since theres some calls to upload before the versioning

Reply:
I see. Can we rename this function to `request_rs_review()` because this is only for remote settings.

Reply:
done!